catalog/transformer/catalog-transformer-pptx/src/main/java/ddf/catalog/transformer/input/pptx/PptxInputTransformer.java

```diff
@@ -117,6 +123,11 @@ private Metacard transformLogic(InputStream input, String id)
                     fileBackedOutputStream.asByteSource()
                             .openStream());
 
+            if(!titleAsMetadataTitle) {
+                // Allow the metacard title to be set as the filename
+                metacard.setAttribute(new AttributeImpl(Core.TITLE, (Serializable)null));
```

Comment:
So as a result of setting this to `null` the filename will be used? Can we not set the filename directly and not rely on a side effect?